### PR TITLE
use token factory

### PR DESCRIPTION
**Pull request by Kouprin**

### CODE REVIEW

File: config/index.js [hunk 1]
```diff
@@ -75,7 +75,8 @@ class RainbowConfig {
     const res = this.maybeGetParam(name)
     if (res === null) {
       console.error(`Parameter ${name} must be specified.`)
-      process.exit(1)
+      throw new Error('error')
```
Comment (ailisp) [inline config/index.js:78]: would be better to throw new Error(`Parameter ${name} must be specified.`) and avoid console.error

File: index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const nearAPI = require("near-api-js")
+
+const getConfig = require("./src/get-config");
+const utils = require('./src/utils');
+
+// Re-export to make sure subsequent libraries use the same version.
+// Without this, ppl would use different version of PublicKey class and the tranactions won't serialize.
```
Comment (ailisp) [inline index.js:7]: :+1:

File: src/utils.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    const near = await setupNear(config);
+    const web3 = await setupEth(config);
+    return { near, web3 }
+}
+
+/**
+ * Remove 0x if prepended
+ * @param {String} value value to check and modify
+ * @return {String} string without 0x
+ */
```
Comment (ailisp) [inline src/utils.js:43]: nice doc

Comment (ailisp) [reply src/utils.js:43]: Hmm I think there's no `modify`. Assume it were, then 
```
    if (value.slice(0, 2) === '0x') {
        return value.slice(2);
```
wouldn't work, so value.slice returns new string and didn't modify exist

File: transfer-eth-erc20/to-near.js [hunk 4]
```diff
@@ -330,6 +349,15 @@ class TransferETHERC20ToNear {
     )
 
     if (transferLog.finished === undefined) {
+      // TODO fix before using
+      // Mint tokens first???
+      /*await ethERC20Contract.methods
+        .mint(ethSenderAccount, Number(amount))
+        .send({ from: ethSenderAccount, gas: 5000000 })*/
```
Comment (ailisp) [inline transfer-eth-erc20/to-near.js:356]: I think we don't mint in eth side, and not all erc20 token has mint, instead we use approve+lock

**Review by ailisp — APPROVED** [review]

File: src/utils.js (hunk 3 above)
Comment (Kouprin) [reply src/utils.js:43]: Will be fixed in https://github.com/near/rainbow-bridge-lib/pull/13

File: config/index.js (hunk 1 above)
Comment (Kouprin) [reply config/index.js:78]: Will be fixed in https://github.com/near/rainbow-bridge-lib/pull/13

**Pull request merged**